src/functions-and-arrays.js

```diff
@@ -1,19 +1,65 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(number1, number2){
+  if (number1 < number2) {
+    return number2;
+    
+  } else {
+    return number1;
+  }
+}
+
+console.log(maxOfTwoNumbers(3,4));
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(array){
+  var longestWord = 5;
+for (i=0; i<array.length; i++){
+  if (array[i].length > longestWord) {
+    longestWord = array[i].length;
+   
+  }
+}
+ return longestWord;
+
+}
+console.log(findLongestWord(words));
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(array){
+  let sum = 0;
+  for(i=0; i<array.length; i++){
+    sum+=array[i]
+  }
+  return sum;
+}
+console.log(sumNumbers(numbers));
+
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(array) {
```

Comment:
Here, Jasmine also asks you to return null if the array is empty so include this piece of code:

if (arr.length === 0) {
    return null;
  }